geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -35,6 +35,7 @@ object SimpleFeatureTypes {
 
     val DefaultDtgField       = "geomesa.index.dtg"
     val EnabledIndices        = "geomesa.indices.enabled"
+    val FeatureExpiration     = "geomesa.feature.expiry"
```

Comment:
Is this configured the same way across Redis and Kafka?

Reply:
Nevermind, I see there's a PR to standardize all that.

Reply:
kafka is still conceptually different, because it only expires things from the in-memory cache. The standardization PR is for accumulo and redis.